### PR TITLE
publish pypi + update doc trigger

### PR DESCRIPTION
**Pull request by dinhlongviolin1**

### CODE REVIEW

**Review by FredLL-Avaiga — APPROVED** [review]

**Review by FabienLelaquais — APPROVED** [review]
Looking good!

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+        uses: pypa/gh-action-pypi-publish@release/v1
+        with:
+          user: __token__
+          password: ${{ secrets.PYPI_API_TOKEN }}
+
+
+  test-published-package:
+    timeout-minutes: 30
+    needs: [publish-to-pypi]
+    environment: production
```
Comment (florian-vuillemot) [inline .github/workflows/publish.yml]:
```suggestion
```
Do not need env here.

File: setup.py [hunk 2]
```diff
@@ -67,23 +67,24 @@ def run(self):
...
+    license="Apache License 2.0",
     include_package_data=True,
     keywords="taipy-gui",
     name="taipy-gui",
     packages=find_packages(include=["taipy", "taipy.gui", "taipy.gui.*"]),
     test_suite="tests",
     tests_require=test_requirements,
     url="https://github.com/avaiga/taipy-gui",
-    version="1.0.0.dev",
+    version="1.0.0.dev0",
```
Comment (florian-vuillemot) [inline setup.py:88]:
```suggestion
    version="1.0.0",
```

File: setup.py [hunk 3]
```diff
@@ -67,23 +67,24 @@ def run(self):
...
         "Intended Audience :: Developers",
-        "License :: OSI Approved :: MIT License",
+        "License :: OSI Approved :: Apache Software License",
         "Natural Language :: English",
         "Programming Language :: Python :: 3",
         "Programming Language :: Python :: 3.8",
         "Programming Language :: Python :: 3.9",
     ],
     description="AI Platform for Business Applications.",
+    long_description=readme,
```
Comment (florian-vuillemot) [inline setup.py:77]: We encountered a problem due to the content of our readme.
But I don't know what to put in place :/

File: setup.py [hunk 4]
```diff
@@ -67,23 +67,24 @@ def run(self):
     classifiers=[
         "Development Status :: 2 - Pre-Alpha",
```
Comment (florian-vuillemot) [inline setup.py]:
```suggestion
```

File: setup.py [hunk 5]
```diff
@@ -67,23 +67,24 @@ def run(self):
...
         "Intended Audience :: Developers",
-        "License :: OSI Approved :: MIT License",
+        "License :: OSI Approved :: Apache Software License",
         "Natural Language :: English",
         "Programming Language :: Python :: 3",
         "Programming Language :: Python :: 3.8",
         "Programming Language :: Python :: 3.9",
```
Comment (florian-vuillemot) [inline setup.py:73]:
```suggestion
        "Programming Language :: Python :: 3.9",
        "Programming Language :: Python :: 3.10",
```

File: setup.py [hunk 6]
```diff
@@ -67,23 +67,24 @@ def run(self):
...
     ],
     description="AI Platform for Business Applications.",
```
Comment (florian-vuillemot) [inline setup.py]:
```suggestion
    description="A 360° open-source platform from Python pilots to production-ready web apps.",
```

File: setup.py (hunk 3 above)
Comment (dinhlongviolin1) [reply setup.py:77]: let me just put a blank output for now

File: setup.py (hunk 2 above)
Comment (dinhlongviolin1) [reply setup.py:88]: I will push version 1.0.0 later, I will first push 1.0.0.dev0 to pypi

File: setup.py [hunk 7]
```diff
@@ -65,25 +65,26 @@ def run(self):
...
-        "License :: OSI Approved :: MIT License",
+        "License :: OSI Approved :: Apache Software License",
         "Natural Language :: English",
         "Programming Language :: Python :: 3",
         "Programming Language :: Python :: 3.8",
         "Programming Language :: Python :: 3.9",
+        "Programming Language :: Python :: 3.10",
     ],
-    description="AI Platform for Business Applications.",
+    description="A 360° open-source platform from Python pilots to production-ready web apps.",
```
Comment (florian-vuillemot) [inline setup.py:76]: Hum, after thinking: @FabienLelaquais @dinhlongviolin1 @FredLL-Avaiga do you have something more appropriate in mind?

Comment (dinhlongviolin1) [reply setup.py:76]: I am not sure whether we should have the same description for all packages or a different one for each

File: setup.py (hunk 2 above)
Comment (florian-vuillemot) [reply setup.py:88]: Good idea

File: setup.py [hunk 8]
```diff
@@ -65,25 +65,27 @@ def run(self):
     author_email="taipy.dev@avaiga.com",
```
Comment (florian-vuillemot) [inline setup.py]:
```suggestion
    author_email="dev@taipy.io",
```

**Pull request merged**